test/ecto/migrator_test.exs

```diff
@@ -161,9 +170,11 @@ defmodule Ecto.MigratorTest do
 
   Application.put_env(:ecto_sql, MigrationSourceRepo, [migration_source: "my_schema_migrations"])
 
+  @base_migration 1_000_000
+
   setup do
     Process.put(:migrated_versions, [1, 2, 3])
-    :ok
+    {:ok, migration_number: System.unique_integer([:positive]) + @base_migration}
```

Comment:
Can we not do this change for now? If we are going to do so, then we should change for the whole file. Your call.

Reply:
If you think so then no problem. I have added this because I was using same call for generating unique integer in more than one `test/2` calls.

Reply:
Right, but other tests call `System.unique_integer([:positive])` too. So I would prefer to change all tests at once or none. :)